app/controllers/find/application_controller.rb

```diff
@@ -10,6 +10,10 @@ class ApplicationController < ActionController::Base
     before_action :redirect_to_cycle_has_ended_if_find_is_down
     before_action :redirect_to_maintenance_page_if_flag_is_active
 
+    rescue_from ActiveRecord::RecordNotFound do |_exception|
+      render_not_found
+    end
```

Comment:
Maybe this version

```suggestion
    rescue_from ActiveRecord::RecordNotFound, with: :render_not_found
```



https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from

Reply:
i likey